geomstats/geometry/dirichlet_distributions.py

```diff
@@ -235,6 +239,78 @@ def coefficients(ind_k):
 
         return gs.squeeze(christoffels)
 
+    def jac_christoffels(self, base_point):
```

Comment:
Would it be possible to write `jacobian` in full? Just because we have started to write it in full in other locations in the code, e.g.:
https://github.com/geomstats/geomstats/blob/d895d33c83ba44229c6f8b9b6deebe457b1b5964/geomstats/geometry/special_euclidean.py#L552

Thanks 🙏